apps/src/templates/instructions/codeReviewV2/CodeReviewTimeline.jsx

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+import CodeReviewTimelineElement, {
+  codeReviewTimelineElementType
+} from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineElement';
+import CodeReviewTimelineCommit from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineCommit';
+import CodeReviewTimelineReview from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineReview';
+import {
+  commitShape,
+  reviewShape
+} from '@cdo/apps/templates/instructions/codeReviewV2/shapes';
+
+const dataType = {
+  review: 'review',
+  commit: 'commit'
+};
+
+const CodeReviewTimeline = props => {
```

Comment:
If this is the "primary" component, it'd be great to have a header comment that gives a short description of how it and its children components work together.